src/components/ui/AspectRatio/aspect-ratio.scss

```diff
@@ -2,7 +2,7 @@
   position: relative;
   display: block;
   width: 100%;
-  height: 0;
+  height: 100%;
```

Comment:
Fixed this gap:
![CleanShot 2022-01-19 at 22 59 54](https://user-images.githubusercontent.com/381395/150249383-df9a024e-ad39-4879-9ce2-28ebb920c07c.gif)